### DISCUSSION TITLE
Broken Link in Contributing > Development > Environment > Setup > golangci-lint

### DISCUSSION THREAD

**Discussion by Crowley723** [authelia/authelia#6877] [Documentation Feedback] [answered]
the link for "golangci-lint" is broken. 
It appears the page was changed from "https://golangci-lint.run/usage/install/" to "https://golangci-lint.run/welcome/install/"
The broken link is under https://www.authelia.com/contributing/development/environment/#setup

**Comment by glime-ai** [comment]
Please note this response was created using generative AI leveraging previously available information from the repository. This information may be helpful however it is our strong recommendation that you carefully consider the information before blindly applying it.

---

I'm sorry, I don't think I got that. Could you please provide more details?

This comment was generated by [Glime](https://glimelab.ai/?utm_source=6fsHkcyufjQLaB&utm_medium=github).

**Comment by Crowley723** [comment, marked as answer]
Should be fixed with #6878.

**Comment by james-d-elliott** [comment]
Thanks!
